bill/invoice.go

```diff
@@ -68,13 +62,13 @@ type Invoice struct {
 	// Expenses paid for by the supplier but invoiced directly to the customer.
 	Outlays Outlays `json:"outlays,omitempty" jsonschema:"title=Outlays"`
 
-	// Summary of all the invoice totals, including taxes.
-	Totals *Totals `json:"totals" jsonschema:"title=Totals"`
-
 	Ordering *Ordering `json:"ordering,omitempty" jsonschema:"title=Ordering Details"`
 	Payment  *Payment  `json:"payment,omitempty" jsonschema:"title=Payment Details"`
 	Delivery *Delivery `json:"delivery,omitempty" jsonschema:"title=Delivery Details"`
```

Comment:
Point of curiosity: I guess this means there's no way to specify multiple ship-to addresses on a single invoice?

Reply:
No, it's not possible to define multiple delivery addresses. I suspect however that this isn't a very widely demanded feature. Normally you'd have purchase orders and delivery notes with details about multiple locations, the invoice would just be the end result of those.

The EU EN 16931-1 standard electronic invoicing definition also has this limitation.